llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -6475,18 +6556,20 @@ SDValue AArch64TargetLowering::LowerINTRINSIC_WO_CHAIN(SDValue Op,
     return DAG.getNode(AArch64ISD::USDOT, dl, Op.getValueType(),
                        Op.getOperand(1), Op.getOperand(2), Op.getOperand(3));
   }
+  case Intrinsic::experimental_get_alias_lane_mask:
```

Comment:
This is no longer relevant, because the intrinsic is always lowered to a `EXPERIMENTAL_ALIAS_LANE_MASK` node by SelectionDAGBuilder?

Reply:
That's correct, removed now.